compiler/src/iree/compiler/ConstEval/Runtime.cpp

```diff
@@ -260,6 +308,52 @@ LogicalResult FunctionCall::addArgument(Location loc, Attribute attr) {
 
     return handleRuntimeError(
         loc, iree_vm_list_push_ref_move(inputs.get(), std::move(bv)));
+  } else if (auto integerAttr = llvm::dyn_cast<IntegerAttr>(attr)) {
+    iree_vm_value_t value;
+    APInt apValue = integerAttr.getValue();
+    switch (apValue.getBitWidth()) {
+    case 8:
+      value =
+          iree_vm_value_make_i8(static_cast<uint8_t>(apValue.getZExtValue()));
+      break;
+    case 16:
+      value =
+          iree_vm_value_make_i16(static_cast<uint16_t>(apValue.getZExtValue()));
+      break;
+    case 32:
+      value =
+          iree_vm_value_make_i32(static_cast<uint32_t>(apValue.getZExtValue()));
+      break;
+    case 64:
+      value =
+          iree_vm_value_make_i64(static_cast<uint64_t>(apValue.getZExtValue()));
+      break;
+    default:
+      return emitError(loc) << "internal error: unsupported consteval jit "
+                               "function integer input type ("
+                            << attr << ")";
+    }
+    return handleRuntimeError(loc,
+                              iree_vm_list_push_value(inputs.get(), &value));
+  } else if (auto floatAttr = llvm::dyn_cast<FloatAttr>(attr)) {
+    iree_vm_value_t value;
+    APFloat apValue = floatAttr.getValue();
+    switch (APFloat::SemanticsToEnum(apValue.getSemantics())) {
+    // case APFloat::S_IEEEhalf:
+    // case APFloat::S_BFloat:
+    case APFloat::S_IEEEsingle:
+      value = iree_vm_value_make_f32(apValue.convertToFloat());
+      break;
+    case APFloat::S_IEEEdouble:
+      value = iree_vm_value_make_f64(apValue.convertToDouble());
+      break;
+    default:
```

Comment:
useful to call out unhandled things but can get compiler error/lsc renaming niceness by having them in code:
```suggestion
    case APFloat::S_IEEEsingle:
      value = iree_vm_value_make_f32(apValue.convertToFloat());
      break;
    case APFloat::S_IEEEdouble:
      value = iree_vm_value_make_f64(apValue.convertToDouble());
      break;
    default:
    case APFloat::S_IEEEhalf:
    case APFloat::S_BFloat:
```

Reply:
Left a long-hand comment instead: there is a long list of semantics types that we will not support at this level.